src/app/(proper_react)/redesign/(authenticated)/user/welcome/EnterInfo.tsx

```diff
@@ -126,7 +126,9 @@ export const EnterInfo = ({ onScanStarted, onGoBack }: Props) => {
       label: l10n.getString("onboarding-enter-details-label-date-of-birth"),
       key: "dateOfBirth",
       type: "date",
-      placeholder: "",
+      placeholder: l10n.getString(
+        "onboarding-enter-details-placeholder-date-of-birth"
+      ),
```

Comment:
Oh, actually, setting a placeholder explicitly removes the placeholder that Firefox adds by default for date elements:

![image](https://github.com/mozilla/blurts-server/assets/4251/042413fc-27c5-4608-aca3-be3d590e255d)

So I think maybe we should just do without the placeholder, or alternatively see if we can do feature detection?

Reply:
Oh. So I guess as already mentioned not showing the placeholder on iOS (which it already decides not to show) would be the easiest and just fix the visual representation of that input.